stl/inc/format

```diff
@@ -1187,6 +1187,24 @@ public:
         }
     }
 
+    _NODISCARD size_t _Estimate_required_capacity() const noexcept {
+        using _CharT        = typename _Context::char_type;
+        size_t _Result      = 0;
+        const auto _Visitor = [&_Result]<class _ArgTy>(const _ArgTy _Arg) noexcept {
+            if constexpr (is_same_v<_ArgTy, basic_string_view<_CharT>>) {
+                _Result += _Arg.size();
+            } else if constexpr (is_same_v<_ArgTy, const _CharT*>) {
+                _Result += char_traits<_CharT>::length(_Arg);
+            } else {
+                _Result += 8;
+            }
+        };
+        for (size_t _Idx = 0; _Idx < _Num_args; ++_Idx) {
+            _STD visit_format_arg(_Visitor, get(_Idx));

```

Comment:
`get` is too pretty (i.e., it's missing `_STD` qualification).

Reply:
It's the member function `get`

Reply:
This is why I shouldn't comment before finishing my coffee - there's no overload of `std::get` for `size_t`, Casey.